daonode/src/main/java/bisq/daoNode/error/StatusException.java

```diff
@@ -46,6 +46,7 @@ public StatusException(Response.Status httpStatus, String message) {
     public static class StatusExceptionMapper implements ExceptionMapper<StatusException> {
         @Override
         public Response toResponse(StatusException exception) {
+            log.error("", exception);
```

Comment:
If you do it this way, you might want to do same in line 33 of `CustomExceptionMapper`.